frontend/src/utils/ValidationSchemes.ts

```diff
@@ -9,15 +9,28 @@ export const validationSchema = {
             .required('Enter your new Email')
             .email('This is not email')
         }),
-      validationPassword: 
-        Yup.object({
-            oldPassword: Yup.string()
-            .required('Enter your old password'),
-            newPassword: Yup.string()
-            .required('Enter your new password')
-            .notOneOf([Yup.ref('oldPassword')], 'Old and new passwords must be different'),
-            confirmPassword: Yup.string()
-            .required('Confirm your password')
-            .oneOf([Yup.ref('newPassword')], 'Passwords do not match')
-            }),
-}
\ No newline at end of file
+    validationPassword: 
+    Yup.object({
+        oldPassword: Yup.string()
+        .required('Enter your old password'),
+        newPassword: Yup.string()
+        .required('Enter your new password')
+        .notOneOf([Yup.ref('oldPassword')], 'Old and new passwords must be different'),
+        confirmPassword: Yup.string()
+        .required('Confirm your password')
+        .oneOf([Yup.ref('newPassword')], 'Passwords do not match')
+        }),
+    validationFile: 
+    Yup.object({
+        file: Yup.mixed()
+        .nullable()
+        .notRequired()
+        .test('FILE_SIZE', 'Uploaded file is too big.', 
+            value => !value || (value && value.size <= FILE_SIZE))
+        // .test('FILE_FORMAT', 'Uploaded file has unsupported format.', 
+        //     value => !value || (value && SUPPORTED_FORMATS.includes(value.type)))
```

Comment:
Why validation on file check is commented?

Reply:
corrected, now everything work